keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigController.kt

```diff
@@ -97,5 +107,26 @@ class DeliveryConfigController(
     validator.validate(deliveryConfig)
   }
 
-  private val log by lazy { LoggerFactory.getLogger(javaClass) }
+  @Operation(
+    description = "Imports a delivery config manifest from source control."
+  )
+  @PostMapping(
+    path = ["/import"],
+    produces = [APPLICATION_JSON_VALUE, APPLICATION_YAML_VALUE]
+  )
+  fun import(
+    @RequestParam repoType: String,
+    @RequestParam projectKey: String,
+    @RequestParam repoSlug: String,
+    @RequestParam manifestPath: String,
```

Comment:
That's a good question. There's some history there. It's called a `path` because you can, indeed, have the file under a sub-directory. The catch is that we wanted to enforce the `.netflix` convention for the "root path" of the manifest, and so that is configured in `igor` and it won't look for manifests anywhere else in the source tree in SCM. This was also due to my concerns with just allowing other Spinnaker services to retrieve arbitrary files from source control via `igor`. So what `manifestPath` means is "the relative path to the manifest under the configured base path".

Examples:
- `spinnaker.yml` --> `.netflix/spinnaker.yml`
- `md/spinnaker.yml` --> `.netflix/md/spinnaker.yml`

If you have suggestions on how to improve the naming or anything here, let me know!

Reply:
The only thing I can think of is adding a comment in a function header (e.g., in the `ScmService` interface).

Reply:
Will do. There's comments and user docs around this behavior in other places, FWIW.

Reply:
Addressed in https://github.com/spinnaker/keel/pull/1261/commits/f84e2bf791c50f98b105b79b58318d55c569d7bf